### ISSUE TITLE
"ACCESS_REFUSED - Login was refused using authentication mechanism PLAIN. " by  AmqpDetailsConnectionProvider

### ISSUE THREAD

**Issue by TopSpoofer** [akka/alpakka#761]
version 0.16  code like that:
```
private val queueDeclaration = QueueDeclaration(queueName)
  private val bindingDeclaration = BindingDeclaration(queue = queueName, exchange = exchangeName)
  private val connectionProvider = AmqpDetailsConnectionProvider(
    host = "192.168.9.241",
    5672,
  ).withCredentials(AmqpCredentials(username = "admin", password = "admin"))

  private val amqpSourceConfig: NamedQueueSourceSettings = NamedQueueSourceSettings(
    connectionProvider = connectionProvider,
    queue = queueName
  ).withDeclarations(queueDeclaration, bindingDeclaration)

```

and make an exception :
```
com.rabbitmq.client.AuthenticationFailureException: ACCESS_REFUSED - Login was refused using authentication mechanism PLAIN. For details see the broker logfile.
	at com.rabbitmq.client.impl.AMQConnection.start(AMQConnection.java:351)
	at com.rabbitmq.client.impl.recovery.RecoveryAwareAMQConnectionFactory.newConnection(RecoveryAwareAMQConnectionFactory.java:64)
	at com.rabbitmq.client.impl.recovery.AutorecoveringConnection.init(AutorecoveringConnection.java:99)
	at com.rabbitmq.client.ConnectionFactory.newConnection(ConnectionFactory.java:948)
	at com.rabbitmq.client.ConnectionFactory.newConnection(ConnectionFactory.java:907)
	at com.rabbitmq.client.ConnectionFactory.newConnection(ConnectionFactory.java:784)
	at akka.stream.alpakka.amqp.AmqpDetailsConnectionProvider.get(AmqpConnectionProvider.scala:130)
	at akka.stream.alpakka.amqp.AmqpConnectorLogic.preStart(AmqpConnector.scala:26)
	at akka.stream.alpakka.amqp.AmqpConnectorLogic.preStart$(AmqpConnector.scala:24)
	at akka.stream.alpakka.amqp.AmqpSourceStage$$anon$1.preStart(AmqpSourceStage.scala:49)
	at akka.stream.impl.fusing.GraphInterpreter.init(GraphInterpreter.scala:293)
	at akka.stream.impl.fusing.GraphInterpreterShell.init(ActorGraphInterpreter.scala:553)
	at akka.stream.impl.fusing.ActorGraphInterpreter.tryInit(ActorGraphInterpreter.scala:675)
	at akka.stream.impl.fusing.ActorGraphInterpreter.preStart(ActorGraphInterpreter.scala:723)
	at akka.actor.Actor.aroundPreStart(Actor.scala:528)
	at akka.actor.Actor.aroundPreStart$(Actor.scala:528)
	at akka.stream.impl.fusing.ActorGraphInterpreter.aroundPreStart(ActorGraphInterpreter.scala:666)
	at akka.actor.ActorCell.create(ActorCell.scala:591)
	at akka.actor.ActorCell.invokeAll$1(ActorCell.scala:462)
	at akka.actor.ActorCell.systemInvoke(ActorCell.scala:484)
	at akka.dispatch.Mailbox.processAllSystemMessages(Mailbox.scala:282)
	at akka.dispatch.Mailbox.run(Mailbox.scala:223)
	at akka.dispatch.Mailbox.exec(Mailbox.scala:234)
	at akka.dispatch.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at akka.dispatch.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at akka.dispatch.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at akka.dispatch.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)

```

i make sure that the host, port, account and password is correct ！

**Comment by ennru** [comment]
This was a bug which is fixed in master and will be released soon: https://github.com/akka/alpakka/pull/755

**Comment by ennru** [comment]
Fixed in the v0.17 release now.
